app/src/components/fields/ReadMoreField.tsx

```diff
@@ -39,7 +39,7 @@ export const ReadMoreField: React.FC<IReadMoreFieldProps> = (props) => {
   const persistentTextPortion = sanitizedText.slice(0, truncationIndex);
 
   return (
-    <Box>
+    <>
       <Typography {...TypographyProps}>
```

Comment:
This component already gets `color="textSecondary"` when it is rendered by the parent:
```tsx
<ReadMoreField text={objectives.objectives} maxCharLength={200} TypographyProps={{ color: 'textSecondary' }} />
```

Reply:
The intent here is to allow the parent to style the "read more" Typography component based on the context it's used in